### PR TITLE
Adding dynamic posts to profile

### PR DESCRIPTION
**Pull request by rafaelchimicatti**
Adding dynamic posts to profile

### CODE REVIEW

File: frontend/src/pages/Profile/Profile.tsx [hunk 1]
```diff
@@ -119,6 +185,19 @@ const ProfileScreen = () => {
   };
 
 
+  async function loadAds() {
+    try {
+      const response = await api.get("/posts/all", {
+        headers: {
```
Comment (edu-correia) [inline frontend/src/pages/Profile/Profile.tsx:191]: Is this needed to work? We are already sending the JWT token via cookies

File: frontend/src/pages/Profile/Profile.tsx [hunk 2]
```diff
@@ -70,17 +82,71 @@ const ProfileScreen = () => {
...
+      console.log(e);
+    }
+  }
+
+  async function sendRating(newRating: number) {
+    const id = loggedUser?.id;
+    try {
+      const response = await api.post(`/ratings`, {
+          grade: newRating,
+          reporterUserId: loggedUser?.id,
```
Comment (edu-correia) [inline frontend/src/pages/Profile/Profile.tsx:117]: This doesn't make much sense, maybe the rating system should only exist in the /users/{dynamic_id} page, so we have 2 different users (logged user vs current page user)
If this is a workaround to speed up the code process, just move along with what we have right now. Your call

**Review by edu-correia — COMMENTED** [review]
2 questions

File: frontend/src/pages/Profile/Profile.tsx (hunk 2 above)
Comment (rafaelchimicatti) [reply frontend/src/pages/Profile/Profile.tsx:117]: Agreed. However, I was using it while we didn't have a profile page for other users.

File: frontend/src/pages/Profile/Profile.tsx (hunk 1 above)
Comment (rafaelchimicatti) [reply frontend/src/pages/Profile/Profile.tsx:191]: I could'nt figure out another way to get all posts without the API request.

**Review by juancoelhoo — APPROVED** [review]

**Comment by juancoelhoo** [comment]
Looks good to me.

File: frontend/src/pages/ProfileId/ProfileId.tsx [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+  const [dropdownVisible, setDropdownVisible] = useState(false);
+
+  const toggleDropdown = () => {
+    setDropdownVisible(!dropdownVisible);
+  };
+
+
+  async function loadAds() {
+    try {
+      const response = await api.get("/posts/all", {
```
Comment (edu-correia) [inline frontend/src/pages/ProfileId/ProfileId.tsx:165]: Shouldn't we call "/posts/${id}"?

**Review by edu-correia — COMMENTED** [review]
One question

**Review by edu-correia — APPROVED** [review]
LGTM

**Pull request merged**